docs/en/reference/best-practices.rst

```diff
@@ -54,7 +54,7 @@ Don't use special characters
 
 Avoid using any non-ASCII characters in class, field, table or
 column names. Doctrine itself is not unicode-safe in many places
-and will not be until PHP itself is fully unicode-aware (PHP6).
+and will not be until PHP itself is fully unicode-aware (PHP7).
```

Comment:
PHP 7 is not fully unicode aware either. This sentence is actually a reference to the long-abandonned PHP 6 version, and should be reworded